### ISSUE TITLE
undefined method `underscore' for "Base":String (NoMethodError)

### ISSUE THREAD

**Issue by jelder** [bear-metal/tunemygc#7]
CLI seems to be broken in 1.0.32.

```
$ bundle exec tunemygc -c XXXXXXXXXXXXXXXXXXXX
[TuneMyGC] Rails not detected, loading minimal agent
/app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/spies.rb:6:in `spy': undefined method `underscore' for "Base":String (NoMethodError)
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/spies.rb:10:in `<module:Spies>'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/spies.rb:4:in `<module:TuneMyGc>'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/spies.rb:3:in `<top (required)>'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/interposer.rb:3:in `require'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/interposer.rb:3:in `<top (required)>'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/agent.rb:4:in `require'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc/agent.rb:4:in `<top (required)>'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc.rb:26:in `require'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/lib/tunemygc.rb:26:in `<top (required)>'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/bin/tunemygc:3:in `require'
        from /app/vendor/bundle/ruby/2.2.0/gems/tunemygc-1.0.32/bin/tunemygc:3:in `<top (required)>'
        from /app/vendor/bundle/ruby/2.2.0/bin/tunemygc:23:in `load'
        from /app/vendor/bundle/ruby/2.2.0/bin/tunemygc:23:in `<main>'
```

**Comment by gravityrail** [comment]
I'm seeing this error too.

**Comment by methodmissing** [comment]
Fixed in 1.0.33 , mind updating and taking it for a spin? Last week we removed activesupport as a runtime dependency, but failed to handle the "manual" pure Ruby GC spy properly.
